package.json

```
@@ -236,7 +236,7 @@
 		"url-polyfill": "^1.1.5",
 		"uuid": "^3.3.2",
 		"vis": "^4.21.0",
-		"webdav": "^2.0.0",
+		"webdav": "ubarsaiyan/webdav-client#custom-request",
```

```suggestion
		"webdav": "^2.10.0",
```

The changes on this custom branch were already merged in the main package https://github.com/perry-mitchell/webdav-client/pull/174